src/System.IdentityModel.Tokens.Jwt/JwtHeader.cs

```diff
@@ -71,6 +79,45 @@ public JwtHeader(SigningCredentials signingCredentials)
         }
 
         /// <summary>
+        /// Gets or sets the <see cref="IDictionary{TKey, TValue}"/> used to map
+        /// the JWA identifiers with the corresponding security algorithms.
+        /// </summary>
+        public static IDictionary<string, string> InboundAlgorithmMap { get; } = new Dictionary<string, string>
```

Comment:
@brentschmaltz @PinpointTownes statics have caused issues in past, thats why we moved all claim mapping maps from static to instance. Are we introducing a static map again?